scripts/update-changelog.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs')
+
+const lerna = require('../lerna')
+const {CHANGELOG_FILENAME} = require('./constants')
+
+const content = fs.readFileSync(`../${CHANGELOG_FILENAME}`, 'utf-8')
+
+const lines = content
+  .split('\n')
+  .map(line => {
+    if (line.startsWith(`## ${lerna.version} (unreleased)`)) {
```

Comment:
This script is expecting the changelog to be like this for the unreleased version:

```md
## v10.0.0-alpha.9 (unreleased)
```

Is this fine with you @kof or should we use instead of the version `Next` maybe?

Reply:
I think unreleased is better because, because next is also used in dist-tag and basically means alpha for us

Reply:
No, unreleased is the place holder for the date. I was asking about replacing the version with `Next`

Reply:
We could have it like this:

```
## Next
```

And then the script could replace it with the version and the date.